wandb/integration/openai/resolver.py

```diff
@@ -1,21 +1,56 @@
 import io
 import logging
+import uuid
 from typing import Any, Dict, List, Optional
 
+import wandb
 from wandb.sdk.data_types import trace_tree
 from wandb.sdk.integration_utils.llm import Response
 
 logger = logging.getLogger(__name__)
 
 
+def get_alias_and_cost(model_name: str, is_completion: bool = False) -> float:
+    alias_cost_mapping = {
```

Comment:
A few thoughts on this:
- This should be auto-generated and imported here. See https://github.com/wandb/wandb/blob/main/tools/generate-tool.py (we use it to generate code in a number of cases) for inspiration. 
- There should be a (nightly) CI action that checks these costs (should find a way to query OpenAI) and posts to a slack channel (https://github.com/wandb/wandb/blob/main/.circleci/config.yml#L1232) in case they changed.
  - Better still, would be great to automatically open a pr with the updated costs in case they change
- Should add a (date) tag (and log it!) to reference these costs in the future as they are likely to change.

Reply:
Makes sense @morganmcg1. Thinking out loud below.
- Let's then at least add the tag for versioning?
- What I'm worried about is if they change the costs, we could potentially account for that in the ui by applying some sort of a transformation (the backend would need to know the latest tag) so that the users of older sdk versions still see accurate cost estimates in the app. This is definitely out of scope for this pr.
- Are these the same for everyone? I think we should consider adding a way to customize these figures (which would provide an additional way to mitigate the situation in the previous point). May be in scope for this pr (or the next one)

Reply:
For the current PR. I've removed the cost aspect completely. Will find a way to add it in an upcoming PR.

Reply:
I additionally removed the model name aliases.